src/SprykerEco/Yves/Payone/Dependency/Client/PayoneToShipmentBridge.php

```diff
@@ -31,6 +31,20 @@ public function __construct($shipmentClient)
      */
     public function getAvailableMethods(QuoteTransfer $quoteTransfer)
     {
+        if (method_exists($this->shipmentClient, 'getAvailableMethodsByShipment') === true) {
+
+            $shipmentMethodsCollectionTransfer = $this->shipmentClient->getAvailableMethodsByShipment($quoteTransfer);
+
+            if ($shipmentMethodsCollectionTransfer->getShipmentMethods()->count() > 1) {
+                throw new \RuntimeException('Split shipping is not supported');
```

Comment:
There should be specific exception.

Reply:
Already discussed with @dereuromark and @tamasnyulas. Recap left RuntimeException for now